index.html

```diff
@@ -48,10 +53,49 @@ <h3>Let's CONNECT</h3>
                 </div>
             </div>
         </section>
+    </section>
         <section class="portofolio">
             <div class="work-container">
-                <div class="port-image"><img src="assets/work1.png" alt=""></div>
-                <div class="work-title"><h1>Tonic</h1></div>
+                <div class="work-container-wrap">
+                <div class="image-info">
+                <div class="port-image">
+                    <img src="assets/work1.png" alt="" class="img-work">
+                    </div>
+                    </div>
+                    <div class="description-work">
+                <div class="work-title"><h1 class="work-header">Tonic</h1></div>
+                <div class="work-technology">
+                    <ul>
+                        <li class="">CANOPY</li>
+                        <li class=""><img src="assets/counter.png" alt=""></li>
+                        <li class="tech">Back End Dev</li>
+                        <li class=""><img src="assets/Counter.png" alt=""></li>
+                        <li class="date">2015</li>
+                    </ul>
+                </div>
+                <div class="work-details">
+                    <p class="work-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-skills">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-button"><button>See Project</button></div>
+                </div>
+                </div>
+            </div>
+            <div class="desktop-work">
```

Comment:
### OPTIONAL

- [ ] Kindly adjust the images and description in  cards for your desktop-work section to look like the Figma design, Each card is divided half of it contains the project's picture and the other half contains the project's description


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_39](https://user-images.githubusercontent.com/69638013/161390384-ef70012b-f477-48a0-ac89-19b66b110266.png)  | ![Screenshot_38](https://user-images.githubusercontent.com/69638013/161390387-ac8707cc-069e-4815-a701-a4d61807bce7.png)